### ISSUE TITLE
Problems replicating data from NAV to BC when using Traefik

### ISSUE THREAD

**Issue by Bjarke848** [microsoft/navcontainerhelper#2916]
PLEASE DO NOT INCLUDE ANY PASSWORDS OR TOKENS IN YOUR ISSUE!!!

**Describe the issue**
We face some issues replicating data from our NAV18 to BC 21.2 using an ISV solution.
This has been workin as inteeded earlier when using BC 20.1 with Traefik  "tobiasfenster/traefik-for-windows:v1.7.34", after upgrade to BC 21.2 this stopped working.
We have tried to make a similar setup with local DB (outside container) and BC 21.2 but without Traefik, and this is working as intended.

One thing we noticed is when trying to create new Traefik container with (Setup-TraefikContainerForBcContainers) it always uses version 1.7.34 even when newer version is available for tobiasfenster/traefik-for-windows.

**Scripts used to create container and cause the issue**
```
Setup-TraefikContainerForBcContainers -PublicDnsName bc.kvik.com -CrtFile "C:\crtfile.crt" -CrtKeyFile "C:\pemfile.pem" -IP **.*.*.** -Recreate
```

**Screenshots**
![image](https://user-images.githubusercontent.com/57141525/220070740-39064c6e-b6c6-46cb-8754-ba80bc150cd8.png)

**Additional context**
- does it happen all the time?
 Yes it does happend all of the time
- did it use to work?
 Is used to work in earlier versions of BC (20.1) with traefik

**Comment by freddydk** [comment]
@tfenster do you know why this might happen?

**Comment by JeppeBylov** [comment]
@tfenster 
The replication is made as a httpwebrequest sending an XML header and body through SOAP, using basic authentication with username/webservicekey.

**Comment by tfenster** [comment]
OK, how big are those files? And has that changed between 20 and 21? Because there is a (configurable) size limit in Traefik, maybe that is the reason

**Comment by JeppeBylov** [comment]
We've seen instability meaning the higher the file size the more errors. And seems to have hit a dead end where even file sizes with 1 record file size should be a few KB's are also failing.

As far as i can see the default maxRequestBodySize is 0, but maybe we should investigate further.

**Comment by freddydk** [comment]
Was this issue resolved?

**Comment by freddydk** [comment]
This might very well be the same problem as https://github.com/microsoft/navcontainerhelper/issues/2945 - as this problem surfaced in 21.
Will close this one, problem should be solved in 21.7
